web/v1/bigquery/sql-runner/playbooks/standard/01-base/01-base-main.yml.tmpl

```diff
@@ -0,0 +1,69 @@
+:targets:
+- :name:
+  :type:    bigquery
+  :project:
+  :region:
+:variables:
+  :model_version:      bigquery/web/1.0.0
+  :input_schema:       rt_pipeline_prod1
+  :scratch_schema:     scratch_prod1
+  :output_schema:      derived_prod1
+  :entropy:            ""
+  :stage_next:         true
+  :start_date:         2020-01-01
+  :lookback_window:
+  :days_late_allowed:
+  :session_lookback:
+  :update_cadence:     7
```

Comment:
Yeah it's in days. We could do that - it's `:update_cadence:` in the already released Redshift model, so I'd suggest that if we're changing here we also change in upcoming Redshift minor release.

Reply:
Question here - do you think this just should be clearer in general, or is it because there's a change from a previous version of the model?

The reason I ask is because `lookback_window` is in hours. `session_lookback` is in days. These should be clarified too I feel?

Reply:
Missed this comment earlier. Yeah, clarify them all!